.gitignore

```diff
@@ -0,0 +1,8 @@
+/composer.lock
+/composer.phar
+/phpunit.xml
+/.phpunit.result.cache
+/phpunit.phar
+/config/Migrations/schema-dump-default.lock
+/vendor/
+/.idea/
```

Comment:
We can discuss in the future, whether we want to add _some_ PhpStorm IDE configuration files. But for now it's OK.